persist/fs/manager.go

```diff
@@ -23,15 +23,34 @@ package fs
 import (
 	"time"
 
+	"github.com/m3db/m3db/clock"
 	"github.com/m3db/m3db/persist"
 	"github.com/m3db/m3db/ts"
 )
 
-// persistManager is responsible for persisting series segments onto local filesystem
+const (
+	bytesPerMb = 1024 * 1024 / 8
+)
+
+var (
+	zeroTime time.Time
```

Comment:
Every other package we've named this `timeZero` I believe.  Perhaps we can use this name instead for consistency?